server/app/controllers/applicant/ApplicantProgramsController.java

```diff
@@ -148,8 +152,37 @@ public CompletionStage<Result> indexWithApplicantId(
             });
   }
 
-  @Secure
```

Comment:
Had to remove the `@Secure` notation so pac4j wouldn't expect a profile and thrown an unauthorized error.